spec/controllers/spree/amazon_controller_spec.rb

```diff
@@ -133,41 +133,43 @@
   end
 
   describe "POST #complete" do
+    let(:order) { create(:order_with_line_items, state: 'confirm') }
```

Comment:
Can we please keep the order creation inline with the tests? I know it's less DRY but I prefer the tests to be as explicit as possible, see: https://robots.thoughtbot.com/lets-not